service-commands/scripts/A.py

```diff
@@ -0,0 +1,243 @@
+            subprocess.run(
+                ["ssh", host, "cat > $MANIFESTS_PATH/config.yaml"],
+                input=open(config).read(),
+            )
+        else:
+            print("Warning no config specified")
+
+    if service == "creator-node":
+        print("Starting creator-node...")
+        subprocess.run(
+            [
+                "ssh",
+                host,
+                "audius-cli launch creator-node --configure-ipfs",
+            ]
+        )
+    elif service == "discovery-provider":
+        print("Starting discovery-provider...")
+        subprocess.run(
+            [
+                "ssh",
+                host,
+                "audius-cli launch discovery --seed-job --configure-ipfs",
+            ]
+        )
+    elif service == "remote-dev":
+        subprocess.run(
+            [
+                "ssh",
+                host,
+                (
+                    "sudo apt update && "
+                    "sudo apt install apt-transport-https ca-certificates curl software-properties-common build-essential python-is-python2 git-secrets && "
+                    "curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add - && "
+                    "sudo sudo add-apt-repository 'deb [arch=amd64] https://download.docker.com/linux/ubuntu focal stable' && "
+                    "sudo apt update && "
+                    "sudo apt install docker-ce && "
+                    "sudo usermod -aG docker $USER && "
+                    'sudo curl -L "https://github.com/docker/compose/releases/download/1.27.4/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && '
+                    "sudo chmod +x /usr/local/bin/docker-compose && "
+                    "sudo chown $USER /etc/hosts && "
+                    "curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash && "
+                    "echo 'export PROTOCOL_DIR=$HOME/audius-protocol' >> ~/.profile && "
+                    "source ~/.nvm/nvm.sh && "
+                    "source ~/.profile && "
+                    "source ~/.bashrc && "
+                    "nvm install 10.23.0 && "
+                    "git clone https://github.com/AudiusProject/audius-protocol.git; "
+                    "git clone https://github.com/AudiusProject/audius-client.git; "
+                    "cd audius-protocol/service-commands && "
+                    "npm install && "
+                    "node scripts/hosts.js add && "
+                    "node scripts/setup.js run init-repos up && "
+                    "cd ../libs && "
+                    "npm link && "
+                    "cd ../service-commands && "
+                    "npm link @audius/libs && "
+                    "cd ~/audius-client && "
+                    "npm install && "
+                    "npm link @audius/libs && "
+                    "echo 'Rebooting machine...' && "
+                    "reboot"
+                ),
+            ]
```

Comment:
Can we have this script run (gated with y/n) service-commands/hosts.js to add the new remote ip?

Reply:
Also here we should print to have user copy over git ssh key or even better if they could enter the file name it scp's it to the remote dev box